dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -178,6 +183,7 @@ public SortedSet<String> getSitemap() {
         SortedSet<String> urls = new TreeSet<>();
         urls.addAll(getToolPaths());
         urls.addAll(getBioWorkflowPaths());
+        urls.addAll(getAppToolPaths());
```

Comment:
Thanks for noticing and handling the sitemap at the same time!
Could use a test as well